core/dbt/include/global_project/macros/materializations/models/materialized_view/create_materialized_view_as.sql

```diff
@@ -0,0 +1,11 @@
+-- MATERIALIZED_VIEW IMPLMENTATION
+{% macro create_materialized_view_as(relation, sql, config) %}
+    {{ adapter.dispatch('create_materialized_view_as',(relation, sql, config)) }}
```

Comment:
will we need to keep up the `macro_namespace convention from the experimental repo?`  {{ return(adapter.dispatch('create_materialized_view_as', macro_namespace = 'dbt_labs_materialized_views')(relation, sql, config)) }}